.travis.yml

```diff
@@ -14,6 +14,14 @@ matrix:
     env: CACHE_NAME=SWAGGER
   - php: 7.2
     env: SWAGGER_VERSION=3.0 CACHE_NAME=OPEN_API REPORT_TESTS_COVERAGE=1
+  - php: 7.3
+    env: CACHE_NAME=SWAGGER
+  - php: 7.3
+    env: SWAGGER_VERSION=3.0 CACHE_NAME=OPEN_API REPORT_TESTS_COVERAGE=1
+  - php: 7.4
+    env: CACHE_NAME=SWAGGER
+  - php: 7.4
+    env: SWAGGER_VERSION=3.0 CACHE_NAME=OPEN_API REPORT_TESTS_COVERAGE=1
```

Comment:
Only one php version should run with `REPORT_TESTS_COVERAGE=1`

Reply:
@DarkaOnLine so which one? 7.4?

Reply:
Have you tested this package on 7.4? If so and it works - let's do 7.4 ;)

Reply:
@DarkaOnLine well, I haven't but I try to test in a few days